src/content-tags-drawer/ContentTagsDrawer.jsx

```diff
@@ -121,6 +122,29 @@ const ContentTagsDrawer = ({ id, onClose }) => {
                 </div>
               ))
               : <Loading />}
+            {otherTaxonomies.length !== 0 && (
+              <div>
+                <p className="h4 text-gray-500 font-weight-bold">
+                  {intl.formatMessage(messages.otherTagsHeader)}
+                </p>
+                <p className="h6 text-gray-500">
+                  {intl.formatMessage(messages.otherTagsDescription)}
```

Comment:
Nit: This shouldn't be `h6` because that's making the text bold. The text should not be bold.

![Screenshot 2024-05-08 at 10 37 18 AM](https://github.com/openedx/frontend-app-course-authoring/assets/945577/0d2c71c7-ffc3-4464-9e95-a6e14a2cbf5b)

vs.

![Screenshot 2024-05-08 at 10 38 16 AM](https://github.com/openedx/frontend-app-course-authoring/assets/945577/7f2e6e0e-47a2-4502-a85f-b81ee0075577)

Reply:
Updated here: https://github.com/openedx/frontend-app-course-authoring/pull/987/commits/5a454164d53ec958805e1c523291aae94f7493a8

Reply:
Thanks!